[new product] add apache-ant (#6162)
captn3m0/endoflife.date: products/apache-ant.md

---
title: Apache Ant
category: app
tags: apache build-tool java-runtime
iconSlug: apacheant
permalink: /ant
alternate_urls:
-   /apache-ant
changelogTemplate: https://ant.apache.org/antnews.html
versionCommand: ant -version
releaseDateColumn: true

identifiers:

> -   cpe: cpe:/a:apache:ant
> -   cpe: cpe:2.3:a:apache:ant

Reviewer: https://repology.org/project/ant/information seems to be the same:

```suggestion
-   cpe: cpe:/a:apache:ant
-   cpe: cpe:2.3:a:apache:ant
-   repology: ant
```

auto:
  methods:
  -   maven: org.apache.ant/ant

releases:
-   releaseCycle: "1.10"
    releaseDate: 2016-12-31
    eol: false
    latest: "1.10.15"
    latestReleaseDate: 2024-08-29

-   releaseCycle: "1.9"
    releaseDate: 2013-03-07
    eol: 2024-06-19
    latest: "1.9.16"
    latestReleaseDate: 2021-07-13

-   releaseCycle: "1.8"
    releaseDate: 2010-02-08

>     eol: true

Reviewer: Setting the EOL(1.8) to releaseDate(1.9) seems reasonable here, since a formal announcement wasn't made, and 1.9 was released a year after the last release on 1.8.

```suggestion
    eol: 2013-03-07
```

    latest: "1.8.4"
    latestReleaseDate: 2012-03-23

---

> > [Apache Ant](https://ant.apache.org/) is a Java library and command-line tool designed to drive processes described in build files as targets and extension points that depend on each other. The primary known usage of Ant is the building of Java applications. Ant supplies a number of built-in tasks that allow you to compile, assemble, test, and run Java applications. Additionally, Ant can be effectively used to build non-Java applications, such as C or C++ applications. More generally, Ant can be used to manage any type of process that can be described in terms of targets and tasks.

Reviewer: A bit too verbose. This section is meant for existing Ant users, so a small summary should work:

```suggestion
> [Apache Ant](https://ant.apache.org/) is a Java library and command-line build tool primarily used for building of Java applications.

Ant does not have a strict release cadence or support policy, but End-of-life is decided by a vote on the [Ant PMC](https://ant.apache.org/contributors.html) and announced.
```

---
title: Apache Ant
category: app
tags: apache build-tool java-runtime
iconSlug: apacheant
permalink: /ant
alternate_urls:
-   /apache-ant
changelogTemplate: https://ant.apache.org/antnews.html
versionCommand: ant -version
releaseDateColumn: true

identifiers:
-   cpe: cpe:/a:apache:ant
-   cpe: cpe:2.3:a:apache:ant
-   repology: ant

auto:
  methods:
  -   maven: org.apache.ant/ant

releases:
-   releaseCycle: "1.10"
    releaseDate: 2016-12-31
    eol: false
    latest: "1.10.15"
    latestReleaseDate: 2024-08-29

-   releaseCycle: "1.9"
    releaseDate: 2013-03-07
    eol: 2024-06-19
    latest: "1.9.16"
    latestReleaseDate: 2021-07-13

-   releaseCycle: "1.8"
    releaseDate: 2010-02-08

>     eol: 2013-03-07

Reviewer 2: ```suggestion
    eol: 2013-03-07 # no announcement, using 1.9 release date
```

    latest: "1.8.4"

>     latestReleaseDate: 2012-03-23

Reviewer 2: ```suggestion
    latestReleaseDate: 2012-05-23
```

According to https://ant.apache.org/antnews.html.

---

> [Apache Ant](https://ant.apache.org/) is a Java library and command-line build tool primarily used for building of Java applications.

> Ant does not have a strict release cadence or support policy, but End-of-life is decided by a vote on the [Ant PMC](https://ant.apache.org/contributors.html) and announced.

Reviewer 2: ```suggestion
Ant does not have a strict release cadence or support policy, but End-of-life is decided by a vote on the [Ant PMC](https://ant.apache.org/contributors.html) and announced in the project's news.
```

WDYT ?
